public/content/lessons/2019/windmills/index.mdx

```diff
@@ -1,9 +1,232 @@
+
+> Show that we can choose a point P in S and a line l going through P such that the resulting windmill uses each point of S as a pivot infinitely many times.
+
+<Figure
+    image="intro_img5.png"
+    video="intro_img5.mp4"
+    show="video"
+/>
+
+## Why it’s “unusual”
+
+Depending on your tolerance of puzzles for puzzles’ sake, you might wonder why anyone would care about such a question. There’s a very good reason: I’d argue that the act of solving this will make you better at math and other related fields, which I’ll explain once you’ve seen the solution.
+
+But certainly on the surface this feels disconnected from other parts of math. Other Olympiad problems involve some function to analyze, or a numerical pattern to deduce; or perhaps a difficult counting setup or an elaborate geometric construction, but problem 2 is an unusually pure puzzle. In some ways, that’s its charm: proving that some initial condition will result in this windmill hitting all points infinitely many times doesn’t test if you know a particular theorem, it tests if you can find a clever perspective.
+
+But that blade cuts both ways; without resting existing results from math, what could someone possibly study that would prepare them to solve this? In fact, this brings us to the second unusual thing about this problem: Based on the results, I’m guessing it turned out to be much harder than the contest organizers expected.
+
+You see, typically, the three problems in each day are supposed to get progressively harder. They’re all hard, of course, but problems 1 and 4 should be doable, problems 2 and 5 should be challenging, and problems 3 and 6...well they can be brutal.
+
+But take a look at how many of the 563 participants that year got perfect scores on each problem: Only 22 got a perfect score for this question number 2. By contrast, 170 got a perfect score on problem 5, which is supposed to be about the same difficulty, and more than twice as many, got a perfect score for problem 3, which is supposed to be harder.
+
+You might notice that only six students got full points for problem 6 that year, so by some measure that was the hardest problem on the test. In fact, the way I introduced things earlier was a little disingenuous, the full data would suggest problem 6 was the real clencher. But what’s strange is if you look at the results of the only six students to solve problem 6, all of whom are clearly phenomenal, world-class problem solvers, this windmill puzzle evaded five out of six of them.
+
+## Trying things out
+
+But again, this problem is hard not hard because of the background knowledge it demands, it asks only for insight. So how do you approach something like this?
```

Comment:
```suggestion
Again, this problem is not hard because of the background knowledge it demands. It asks only for insight. So how do you approach something like this?
```